migration-guide.md

```diff
@@ -0,0 +1,65 @@
+# Migrating from v5 to v6
+
+### Upstream Changes
+We have moved the upstream target from [v4.67.0](https://github.com/pulumi/pulumi-aws/pull/2521) to targeting v5.8.0. That means that the following `CHANGELOG`s are relevant:
+- [v5.0.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.0.0)
+- [v5.0.1](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.0.1)
+- [v5.1.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.1.0)
+- [v5.2.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.2.0)
+- [v5.3.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.3.0)
+- [v5.4.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.4.0)
+- [v5.5.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.5.0)
+- [v5.6.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.6.0)
+- [v5.6.1](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.6.1)
+- [v5.6.2](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.6.2)
+- [v5.7.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.7.0)
+- [v5.8.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.8.0)
+
+### Removing depreciated resources
```

Comment:
Are these resources fully removed or have they been aliased? From a user's perspective, what changes do they need to make?

Reply:
They have been removed, in that they are no longer generated as resources and functions in downstream SDKs. They are type aliased in the schema, so users will not need to perform state surgery to continue when they move to the new definitions.